src/_includes/_layouts/Projects.jsx

```diff
@@ -11,7 +11,39 @@ export default ({ comp }) => (
       <div>
         <comp.Navbar />
         <comp.Projects.Hero />
-        <comp.Projects.Projects />
+        <section className="flex flex-col items-center justify-center pt-20 pb-10 gap-y-6 bg-white">
```

Comment:
On my version it is showing up like this:

![image](https://github.com/user-attachments/assets/618c50a6-88d0-47b6-947d-663cd10eb94a)

On the screenshot it looks correct, but just wanted to flag this